cfy_manager/components/patchify/patchify.py

```diff
@@ -0,0 +1,51 @@
+#########
+# Copyright (c) 2019 Cloudify Platform Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import errno
+import logging
+from os.path import join, expanduser
+from getpass import getuser
+
+from ..components_constants import SOURCES, SECURITY, MASTER_IP
+from ..base_component import BaseComponent
+from ..service_names import CLI, MANAGER, CLUSTER
+from ...config import config
+from ...logger import (get_logger,
+                       set_file_handlers_level,
+                       get_file_handlers_level)
+from ...utils import common
+from ...constants import EXTERNAL_CERT_PATH
+from ...utils.install import yum_install, yum_remove
+
+logger = get_logger(CLI)
+
+
+class PatchifyComponent(BaseComponent):
+
+    def __init__(self, skip_installation):
+        super(PatchifyComponent, self).__init__(skip_installation)
+
+    def _install(self):
+        source_url = config['patch'][SOURCES]['patch_source_url']
+        yum_install(source_url)
+
+    def remove(self):
+        sources = config['patch'][SOURCES]
+        yum_remove(sources['patch_source_url'])
+
+    def install(self):
+        logger.notice('Installing Patchify...')
+        self._install()
```

Comment:
Why call `self._install()`, instead of just putting the body of `_install()` here?
`_install()` isn't really reusable.